lib/field.gi

```diff
@@ -1329,7 +1329,7 @@ InstallMethod( PreImagesElm,
     [ IsFieldHomomorphism, IsObject ],
     function ( hom, elm )
     if IsInjective( hom ) = 1 then
-      return [ PreImagesRepresentative( hom, elm ) ];
+      return [ PreImagesRepresentativeNC( hom, elm ) ];
```

Comment:
@cdwensley To give an example of what might break if packages are not adapted first: if this function is called on a homomorphism provided by a package, it will, after the changes in this PR, suddenly produce an error, because it invokes `PreImagesRepresentativeNC` yet the package (not having been adapted yet to changes) only provides a method for `PreImagesRepresentativeNC`.

Reply:
Without understanding why it happens, trying out the Semigroups tests has convinced me that errors do occur.  So, following your "other way around", in what way does "perform the renaming here, but for now leave PreImagesRepresentative as a synonym for PreImagesRepresentativeNC (so that methods installed for the one are also installed for the other)" differ from what has been done so far?  Using the alternative 
    DeclareSynonym( "PreImagesRepresentative", PreImagesRepresentativeNC );
the Semigroups tests now pass.  Any further suggestions?

Reply:
Utils now uses PreImagesRepresentativeNC as suggested above.